### PR TITLE
Add API for support multiple classes

### LINKED ISSUE #12: Multiple classes
It will be very useful to have possibility to check multiple classes simultaneously (something like `forClasses`). Now, it's necessary to duplicate code for each class.

### PR DESCRIPTION
**Pull request by nikitin-da**
Closes #12 
@artem-zinnatullin PTAL =)

### CODE REVIEW

**Comment by nikitin-da** [comment]
Oh, self-closing element not allowed.
I'll fix it this evening!

File: checker/src/main/java/com/pushtorefresh/private_constructor_checker/PrivateConstructorChecker.java [hunk 1]
```diff
@@ -85,35 +90,63 @@ public void check() {
...
+        return new Builder(Collections.singletonList(clazz));
+    }
+
+    public static Builder forClasses(Class... classes) {
+        if (classes == null || classes.length == 0) {
+            throw new IllegalArgumentException("classes can not be null or empty");
+        }
+        for (Class clazz : classes) {
+            if (clazz == null) {
+                throw new IllegalArgumentException("class can not be null");
```
Comment (artem-zinnatullin) [inline checker/src/main/java/com/pushtorefresh/private_constructor_checker/PrivateConstructorChecker.java:109]: nice

File: checker/src/main/java/com/pushtorefresh/private_constructor_checker/PrivateConstructorChecker.java [hunk 2]
```diff
@@ -85,35 +90,63 @@ public void check() {
...
+        return new Builder(Collections.unmodifiableList(asList(classes)));
     }
 
     /**
-     * Runs the check which will assert that required class has one private constructor
+     * Runs the check which will assert that all required classes have one private constructor
      * which throws or not throws exception.
      */
     public void check() {
+        for (Class clazz : classes) {
```
Comment (artem-zinnatullin) [inline checker/src/main/java/com/pushtorefresh/private_constructor_checker/PrivateConstructorChecker.java:121]: Easy :)

**Comment by artem-zinnatullin** [comment]
Awesome! :+1:

**Comment by nikitin-da** [comment]
fixed=)

**Comment by artem-zinnatullin** [comment]
Cool, thanks a lot!

**Pull request merged**